recipes/jpy/meta.yaml

```diff
@@ -0,0 +1,100 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "jpy" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/bcdev/jpy.git
```

Comment:
I think this should be updated to https://github.com/jpy-consortium/jpy

Reply:
@devinrsmith thanks so much for noticing it. I have updated it.